python/shotgun_desktop/startup.py

```diff
@@ -654,7 +654,7 @@ def __ensure_engine_compatible_with_qt_version(engine, app_version):
         )
         return
 
-    # Shotgun Desktop 2.5.0 introduced Python 3 and PySide2 support while being backward
+    # Flow Production Tracking Toolkit 2.5.0 introduced Python 3 and PySide2 support while being backward
```

Comment:
Inaccurate information from before. Replace "Flow Production Tracking Toolkit" to "tk-desktop here please